Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -857,6 +858,15 @@ WebIDL::ExceptionOr<void> HTMLElement::set_popover(Optional<String> value)
     return {};
 }
 
+void HTMLElement::adjust_computed_style(CSS::StyleProperties& style)
+{
+    // https://drafts.csswg.org/css-display-3/#unbox
+    if (local_name() == HTML::TagNames::wbr) {
```

Comment:
This doesn't seem great, but I wasn't sure where else this could go given that `<wbr>` doesn't have an `HTMLElement` subclass.